README.md

```diff
@@ -87,7 +87,17 @@ yarn verify --network $NETWORK
 
 This package additionally contains a `networks.json` file at the root with the address of each deployed contract as well the hash of the Ethereum transaction used to create the contract.
 
-## Solver Authentication
+## Test coverage [![Coverage Status](https://coveralls.io/repos/github/gnosis/gp-v2-contracts/badge.svg?branch=main)](https://coveralls.io/github/gnosis/gp-v2-contracts?branch=main)
```

Comment:
Shouldn't the badge go at the top left of the file?

Reply:
Is there a specific spot for it? I placed it there because it's relevant, while at the top it would be a bit distracting.
Anyway, this is a change that has already been merged to main, you can change it with a new PR. I don't particularly mind the position of the badge either way.